File: Programmers/Python/Lv.0/[Lv.0] 피자 나눠 먹기 (1)/[Lv.0] 피자 나눠 먹기 (1)_00.py
def solution(n):
    answer = 0

    # n을 7로 나누기 (피자 7조각)
    # 나머지가 있으면 + 1 (최소 피자 1판 필요)
    if n % 7 == 0:
        answer = n // 7
    else:
        answer = n // 7 + 1

    return answer

File: Programmers/코딩테스트 입문/Java/Lv.0/[Lv.0] 피자 나눠 먹기 (1)/[Lv.0] 피자 나눠 먹기 (1)_00.java
class Solution {
    public int solution(int n) {
        int answer = 0;

        // n을 7로 나누기 (피자 7조각)
        // 나머지가 있을 경우 + 1 (최소 피자 1판 필요)
        if (n % 7 == 0) {
            answer = n / 7;
        } else {
            answer = n / 7 + 1;
        }

        return answer;
    }
}
